apps/errant-tower-client/src/common/styles/mixins.scss

```
@@ -0,0 +1,61 @@
+@use './variables.scss' as *;
+
+@mixin primaryButton {
```

Stylelint (17.9.0)

[error] Expected mixin name "primaryButton" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)